### ISSUE TITLE
Custom Property Attributes Structure

### ISSUE THREAD

**Issue by lvca** [ArcadeData/arcadedb#651]
### Discussed in https://github.com/ArcadeData/arcadedb/discussions/650

<div type='discussions-op-text'>

<sup>Originally posted by **gramian** November 29, 2022</sup>
Dear Developers,

I am experimenting with custom property attributes. When I query the schema of a database with such custom attributes of properties via `SELECT FROM schema:types`, the `custom` key of a type has a `list` (array) of `maps` (key-value-pairs) storing these custom attributes. Now, I wonder why is this a list of maps and not just a plain map (below `custom`)?

Thank you for your insight</div>

**Comment by lvca** [comment]
From @gramian:

@lvca Please reopen this issue:
Unfortunately, in **22.11.1**, custom attributes are still a `list` of `maps` and not a single `map`. Given the code above:
```
CREATE DOCUMENT TYPE mytype;
CREATE PROPERTY mytype.myprop STRING;
ALTER PROPERTY mytype.myprop CUSTOM meta1 = 'how';
ALTER PROPERTY mytype.myprop CUSTOM meta2 = 'are';
ALTER PROPERTY mytype.myprop CUSTOM meta3 = 'you';
```
then invoking:
```
SELECT properties FROM schema:types
```
I get:
```
[{"custom":[{"meta1":"how"},{"meta2":"are"},{"meta3":"you"}],"name":"myprop","id":1,"type":"STRING"}]
```
Thank you for taking another look.

**Comment by lvca** [comment]
Map for custom was only implemented for type custom, not for property custom. Fixed and now will be released in 22.12.1
